playbook-website/app/javascript/components/AvailableProps/globalPropsValues.ts

```diff
@@ -0,0 +1,249 @@
+  },
+  {
+    key: 17,
+    prop: "justifySelf",
+    type: "union",
+    values: '"start" | "end" | "center" | "auto" | "stretch"'
+  },
+  {
+    key: 18,
+    prop: "lineHeight",
+    type: "union",
+    values:
+      '| "loosest" | "looser" | "loose" | "normal" | "tight" | "tighter" | "tightest"'
+  },
+  {
+    key: 19,
+    prop: "marginRight",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 20,
+    prop: "marginLeft",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 21,
+    prop: "marginTop",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 22,
+    prop: "marginBottom",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 23,
+    prop: "marginX",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 24,
+    prop: "marginY",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 25,
+    prop: "margin",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 26,
+    prop: "maxWidth",
+    type: "union",
+    values: '| "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 27,
+    prop: "numberSpacing",
+    type: "union",
+    values: "tabular"
+  },
+  {
+    key: 28,
+    prop: "order",
+    type: "union",
+    values:
+      '"none" | "first" | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12'
+  },
+  {
+    key: 29,
+    prop: "paddingRight",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 30,
+    prop: "paddingLeft",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 31,
+    prop: "paddingTop",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 32,
+    prop: "paddingBottom",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 33,
+    prop: "paddingX",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 35,
+    prop: "paddingY",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 36,
+    prop: "padding",
+    type: "union",
+    values: '"none" | "xxs" | "xs" | "sm" | "md" | "lg" | "xl"'
+  },
+  {
+    key: 37,
+    prop: "position",
+    type: "union",
+    values: '"relative" | "absolute" | "fixed" | "sticky" | "static"'
+  },
+  {
+    key: 38,
+    prop: "shadow",
+    type: "union",
+    values: '"none" | "deep" | "deeper" | "deepest"'
+  },
+  {
+    key: 39,
+    prop: "space",
+    type: "union",
+    values: '"spaceBetween" | "spaceAround" | "spaceEvenly"'
+  },
+  {
+    key: 40,
```

Comment:
Why key? I imagine we could add more props in above and cause this key to be 41 for example.